src/home/fetch-github/fetch-issues-preview.ts

```diff
@@ -71,7 +77,7 @@ export async function fetchIssuePreviews(): Promise<TaskNoFull[]> {
     }
   }
 
-  const tasks = freshIssues.map((preview: GitHubIssue) => ({
+  const tasks = freshIssues.filter((issue: GitHubIssue) => (issue.labels as GitHubLabel[]).some((label) => label.name.includes("Pricing")) == !IS_RFC).map((preview: GitHubIssue) => ({
```

Comment:
How `IS_RFC` gets a value here? I mean that we define it [here](https://github.com/ubiquity/work.ubq.fi/blob/334442e26383130dbc6b32a64a34dfcd72b6deb4/src/home/fetch-github/fetch-issues-preview.ts#L1) but don't assign it a value anywhere. Is it working?

Reply:
env variables are passed in at build time

source: comments at the top of `src/home/rendering/render-github-login-button.ts`